app/javascript/react/components/SauceShowPage.js

```diff
@@ -1,40 +1,60 @@
 import React, { useState, useEffect } from 'react'
 import _ from 'lodash'
+import ReviewTile from './ReviewTile'
 
 const SauceShowPage = (props) => {
     const [sauce, setSauce] = useState({})
-
+  
     useEffect(() => {
         fetchSauce()
     }, [])
 
     const fetchSauce = async () => {
-        try {
-            const response = await fetch(`/api/v1/sauces/${props.match.params.id}`)
-            if (!response.ok) {
-                const errorMessage = `${response.status} (${response.statusText})`
-                const error = new Error (errorMessage)
-                throw(error)
-            }
-            const sauceObject = await response.json()
-            setSauce(sauceObject)
-        } catch (error) {
-            console.error(`Error in fetch: ${error.message}`)
+      try {
+        const response = await fetch(`/api/v1/sauces/${props.match.params.id}`)
+        if (!response.ok) {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error (errorMessage)
+          throw(error)
         }
+        const sauceObject = await response.json()
+        setSauce(sauceObject.sauce)
+      } catch (error) {
+        console.error(`Error in fetch: ${error.message}`)
+      }
     }
-
+    
     if (_.isEmpty(sauce)) {
       return null
     }
+     
+    const reviewTiles = sauce.reviews.map((review)=>{
+      return(
+        <ReviewTile
+        key = {review.id}
+        id = {review.id}
+        title = {review.title}
+        rating = {review.rating}
+        heatIndex = {review.heatIndex}
+        body = {review.body}
+        createdAt = {review.created_at}
```

Comment:
indent in each of these props
also, there shouldnt be a space before or after the `=` in each of the props